src/scss/jsoneditor/_variables.scss

```diff
@@ -9,6 +9,7 @@ $jse-number: #ee422e !default;
 $jse-boolean: #ff8c00 !default;
 $jse-null: #004ed0 !default;
 $jse-invalid: #000000 !default;
+$jse-color-hex: #1A1A1A !default;
```

Comment:
Instead of `#1A1A1A`, can you please use `$jse-content-color` here? Also, maybe we should simply name it `$jse-color` because it's a color not only when it contains a hex value, but but also for color names like `red`.